projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -1165,6 +1185,21 @@ public void updateV1CronJob(
 
   public void deleteCronJob(String name) throws ApiException {
```

Comment:
Again this function wasn't implemented properly to begin with?

Reply:
Correct.